editor/src/templates/project-not-found/ProjectNotFound.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react'
+import PyramidLight404 from './404_pyramid_light.png'
```

Comment:
we have a `resources` folder in the editor, images saved there are served like so `https://github.com/concrete-utopia/utopia/blob/34a6bb591ba1fa30c380c9904d8d87139a27d5a5/editor/src/templates/index.html#L150`. 

I believe the images in `resources` are served with proper caching and other such stuff as opposed to being expensively served by the haskell server.